.github/workflows/helm_charts_scalar.yml

```diff
@@ -183,9 +184,14 @@ jobs:
           kubectl wait --for=condition=complete --timeout=60s job/schema-loading
           kubectl get pods,svc,endpoints,nodes -o wide
 
-      - name: Run chart-testing (install)
+      - name: Run chart-testing (updated chart)
+        if: ${{ github.event_name != 'workflow_dispatch' }}
         run: ct install --config .github/ct.yaml
```

Comment:
If CI is triggered by `push` or `pull_request`, we only check the updated chart.